ZipBuilder/Package.swift

```diff
@@ -26,18 +26,19 @@ let package = Package(
     .executable(name: "UpdateFirebasePod", targets: ["UpdateFirebasePod"]),
   ],
   dependencies: [
+    .package(url: "https://github.com/apple/swift-argument-parser", .upToNextMinor(from: "0.0.1")),
```

Comment:
Do we really want to float the dependency versions?

Reply:
I was going off their comment that says they're committed to source-stability within minor versions at the bottom of this section:

https://forums.swift.org/t/parsing-file-urls-with-argumentparser/34173

I'm happy to make this exact though, I'll do so now.